codelists/forms.py

```diff
@@ -106,6 +107,10 @@ def clean_csv_data(self):
                 ]
             )
 
+        cleaned_header = [h.strip() for h in header]
+        if header != cleaned_header:
+            data = data.replace(",".join(header), ",".join(cleaned_header), 1)
```

Comment:
are all your CSV files guaranteed not to have quoted items? I think this would fail if they do? 
i.e. your test case is
```csv
 code , description
1234, stuff
```
but this would also be a valid csv file:
```csv
" code ", " description"
"1234","stuff"
```
and csv parsers typically parse `" code", " description"` to `[" code", "description"]`, not `["\" code\"", "\"description\""]` - I think the replace would then fail?